generate.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable no-console */
```

Comment:
Moved this code into a separate file so that it can be reused for the various commands. Otherwise, it is unchanged from the previous version.